src/cirrus/build.py

```diff
@@ -45,6 +67,13 @@ def main():
         'venv',
         'bin',
         'virtualenv')
+
+    # remove existing virtual env if building clean
+    if opts.clean is True and os.path.exists(venv_path):
+        cmd = "rm -r {0}".format(venv_path)
+        print "Removing existing virtualenv: {0}".format(venv_path)
```

Comment:
You should use the logger. https://github.com/evansde77/cirrus/blob/b13e908690c1c58b244ce1269f1bfccc62f052ae/src/cirrus/deploy.py#L18

In general one should avoid print (if logging facilities are provided); if you do need to use it then use the functional form `print(...)`.